docker-compose.yml

```diff
@@ -28,13 +28,21 @@ x-logging:
   options:
     max-size: 1m
     max-file: "1"
+x-volumes:
+  &default-volumes
+  - .:/app:cached
+  - bundle:/bundle
+  - node_modules:/app/node_modules
+  - packs:/app/public/packs
+  - rails_cache:/app/tmp/cache
```

Comment:
Should we add `:cached` to any other of these? I'm not quite sure how it works but the article recommends adding it on any generated content.

Reply:
I remember seeing somewhere that `:cached` was used mostly for source code, which is why I followed the article on only having it on `/app`